tests/test_interactive.py

```diff
@@ -6,6 +6,7 @@
 from rasa_core import utils
 from rasa_core.training import interactive
 from rasa_core.utils import EndpointConfig
+from rasa_core.actions import action
```

Comment:
let's import the function
```suggestion
from rasa_core.actions.action import default_actions
```